ci/ray_ci/automation/wheels_pypi.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+if [[ -z "$RAY_HASH" ]]; then
+    echo "RAY_HASH env var should be provided"
+    exit 1
+fi
+
+if [[ -z "$RAY_VERSION" ]]; then
+    echo "RAY_VERSION env var should be provided"
+    exit 1
+fi
+
+TEST_PYPI_TOKEN="test_pypi_token"
+
+download_wheel() {
+    # Download single wheel from given URL into TMP_DIR/<wheel_filename>.whl
+    WHEEL_URL=$1
+    OUTPUT_FILE="${TMP_DIR}/${WHEEL_URL##*/}"
+    if [ "${OVERWRITE-}" == "1" ] || [ ! -f "${OUTPUT_FILE}" ]; then
```

Comment:
nit: `[[ "${OVERWRITE-}" == "1" || ! -f "${OUTPUT_FILE} ]]`

github copilot learned a lot of bad shell script styling..

Reply:
oh I copied this part straight from `release/util/download_wheels.sh`. Will fix this